ibis/expr/api.py

```diff
@@ -191,8 +191,7 @@
 True
 """
 
-
-_ = deferred = Deferred()
+_ = deferred = var("_")
```

Comment:
Can we import this from `ibis.comon.deferred`, rather than calling `var("_")` in two places? Right now it's not clear to me why we create a `_` instance twice.

Reply:
Indeed, updating.